### PR TITLE
Add support for channel auth

### PR DESCRIPTION
**Pull request by gheine**

### CODE REVIEW

File: app/io/flow/play/util/AuthData.scala [hunk 1]
```diff
@@ -500,3 +508,54 @@ object OrgAuthData {
...
+              )
+            )
+          }
+          case _ => None
+        }
+
+      }
+    }
+  }
+}
```
Comment (mikeclimbrock) [inline app/io/flow/play/util/AuthData.scala:561]: nitpick: new line

**Review by mikeclimbrock — COMMENTED** [review]
overall, lgtm, assuming alignment on the updates to token api spec

**Review by mbryzek — APPROVED** [review]

**Comment by gheine** [comment]
@mikeclimbrock does this need alignment on the updates to token api spec? I think it can be merged before/independent of that.

**Review by mikeclimbrock — APPROVED** [review]
given token api changes were reverted, this lgtm

**Pull request merged**